test-system/lobster-python/pytest_mark.output

```diff
@@ -48,4 +48,3 @@ Written output for 2 items to pytest_mark.lobster
   "schema": "lobster-act-trace",
   "version": 3
 }
-
```

Comment:
Is this change really necessary?

Reply:
Previously make test-ci command added two empty lines at the end of the content in output files 
I made this change to remove the extra lines.

In recent commit, I have removed the code which added extra empty line to the end of content in output files,
now you will see only one empty line at the end of content in those output files